File: practice/algorithms/warmup/solve_me_second/SolveMeSecond.scala
object SolveMeSecond extends App {
  val n = readInt
  //recommended
  (1 to n).map(i => readLine.split(" ").map(_.toInt).sum).foreach(println)

  /*
    another way
    for (i <- 1 to n) {
    val Array(a, b) = readLine.split(" ").map(_.toInt)
    println(a + b)
  }
  */
}

File: practice/algorithms/warmup/solve_me_second/SolveMeSecond.java
import java.util.*;

class SolveMeSecond {

    static int solveMeSecond(int a, int b) {
        return a+b;
    }

    public static void main(String[] args) {
        Scanner in = new Scanner(System.in);
        int t;
        int sum;
        int a,b;
        t = in.nextInt();
        for (int i=0;i<t;i++) {
            a = in.nextInt();
            b = in.nextInt();
            sum = solveMeSecond(a, b);
            System.out.println(sum);
        }
    }
}
